ncl/ncl_entries/day_of_week.ipynb

```diff
@@ -0,0 +1,232 @@
+   "metadata": {},
+   "source": [
+    "## Using `cftime`\n",
+    "[`cftime`](https://unidata.github.io/cftime/index.html) offers a solution to these potential issues by supporting year 0 and all of the [CF Convention calendars](https://cfconventions.org/cf-conventions/cf-conventions#calendar). Here is how you can use `cftime` to calculate the day of the week."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Proleptic Gregorian calendar: \t\t2024-6-4 is a Tuesday\n",
+      "Standard calendar: \t\t\t2024-6-4 is a Tuesday\n",
+      "Julian calendar: \t\t\t2024-6-4 is a Monday\n",
+      "360 day calendar: \t\t\t2024-6-4 is a Wednesday\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cftime\n",
+    "\n",
+    "day = 4\n",
+    "month = 6\n",
+    "year = 2024\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='proleptic_gregorian', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"Proleptic Gregorian calendar: \\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='standard', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"Standard calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(year, month, day, calendar='julian', has_year_zero=True).strftime(\n",
+    "    '%A'\n",
+    ")\n",
+    "print(f\"Julian calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='360_day', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"360 day calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Differences between `cftime` and NCL's `day_of_week`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calendars\n",
+    "NCL's `day_of_week` only supports the proleptic Gregorian calendar, while the `cftime` module supports all CF conventions calendars, including the `proleptic_gregorian` calendar."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Type\n",
+    "Notably, using `cftime` to calculate the day of the week works by getting a `dayofwk` value from a `cftime.datetime` object, which means that the calculations for the day of the week have to be collected through each date individually, while NCL's `days_in_month` can take in multidimensional integer arrays, given that the `year` and `month` arrays have the same dimensions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Return Value\n",
+    "There is a slight difference in the resulting value between NCL's `day_of_week` and `cftime`/`datetime`. \n",
```

Comment:
This is with `datetime.date.weekday()` and `cftime.datetime.dayofwk` right as opposed to those objects with `strftime('%w')`?

Maybe this text is from when you had something different above.  I think it's maybe lacking that context now.  

Should be pretty easy to reword though.

Reply:
You are correct that much of that section is left over from a previous version. I am struggling to decide whether or not a `return value` section is even necessary anymore, given that the results are identical with `strftime`. Would it be more beneficial to simply mention this in the `Using cftime` section rather than in its own section?

Reply:
Good point.  I think that makes sense.